### PR TITLE
[bug] Fix failed tests for with-jest example

### PR DESCRIPTION
**Pull request by keeeparis**
Closes the issue https://github.com/vercel/next.js/issues/37082

This PR fixes an issue with jest tests that failed because ```*.module.css``` file was located in ```pages``` directory. File location now changed to ```src/styles``` and import of styles changed accordingly.

### CODE REVIEW

File: examples/with-jest/styles/index.module.css [hunk 1]
```diff
@@ -1,87 +1,88 @@
...
+  margin-left: 0.5rem;
+}
+
+@media (max-width: 600px) {
+  .grid {
+    width: 100%;
+    flex-direction: column;
+  }
+}
+diff --git a/examples/with-jest/pages/index.module.css b/examples/with-jest/pages/index.module.css
```
Comment (Brooooooklyn) [inline examples/with-jest/styles/index.module.css:88]: unnecessary line

Comment (keeeparis) [reply examples/with-jest/styles/index.module.css]: resolved.

**Review by leerob — APPROVED** [review]

**Comment by keeeparis** [comment]
How can I fix failed lint check? It's related to Prettier.

**Review by ijjk — CHANGES_REQUESTED** [review]
Hi, this example is working correctly against the latest canary of Next.js please update and give it a try

**Pull request closed without merging**